spec/rails_helper.rb

```diff
@@ -8,60 +8,28 @@
 require 'rspec/rails'
 require 'capybara/rspec'
 require 'capybara/rails'
-# Add additional requires below this line. Rails is not loaded until this point!
-
-# Requires supporting ruby files with custom matchers and macros, etc, in
-# spec/support/ and its subdirectories. Files matching `spec/**/*_spec.rb` are
-# run as spec files by default. This means that files in spec/support that end
-# in _spec.rb will both be required and run as specs, causing the specs to be
-# run twice. It is recommended that you do not name files matching this glob to
-# end with _spec.rb. You can configure this pattern with the --pattern
-# option on the command line or in ~/.rspec, .rspec or `.rspec-local`.
-#
-# The following line is provided for convenience purposes. It has the downside
-# of increasing the boot-up time by auto-requiring all files in the support
-# directory. Alternatively, in the individual `*_spec.rb` files, manually
-# require only the support files necessary.
 #
 Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
 
-# Checks for pending migration and applies them before tests are run.
-# If you are not using ActiveRecord, you can remove this line.
 ActiveRecord::Migration.maintain_test_schema!
 
+GDS::Metrics.configure do |config|
```

Comment:
Can this be moved to `spec/features/model_metrics_spec.rb` or do we need this setup before every spec?

Reply:
I think it should be in rails_helper as it's part of the environment setup (it doesn't vary with tests). Otherwise, the test environment would change depending on when the associated test file is run.